portfolio/src/main/webapp/index.html

```diff
@@ -1,31 +1,64 @@
 <!DOCTYPE html>
 <html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Giovanni's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
 
-<head>
-	<meta charset="UTF-8">
-	<title>Giovanni's Portfolio</title>
-	<link rel="stylesheet" href="style.css">
-	<script src="script.js"></script>
-</head>
-
-<body>
-	<div id="content">
-		<h1>Giovanni's Portfolio</h1>
-		<p>Hello, I am a Computer Engieering Student, at the University of Florida. </p>
-		<p>Click here to get a random message:</p>
-		<button onclick="randomFact()">click for a random Gio fact.</button>
-		<p>Click <a href="images/images.html">here</a> to see more pictures of me!</p>
-		<p>Click <a href="blogs/blogs.html">here</a> to see what people have written about me!</p>
-		<p>Click <a href="Projects/Projects.html">here</a> to see my projects(🚧Under Construction!🚧)</p>
-
-		<p> <a href="Resume.pdf">RESUME</a>.</p>
-		<p> <a href="https://www.linkedin.com/in/giovanni-ferioli">My LinkedIn</a> </p>
-		<div id="fact-container"></div>
-		<button onclick="getName()">Use fetch()</button>
-		<div id="greeting-container"></div>
-        <div id="list-container"></div>
-	</div>
-
-</body>
+  <body>
+    <div id="content">
+      <h1>Giovanni's Portfolio</h1>
+
+      <p>Hello, I am a Computer Engineering Student, at the University of Florida. </p>
+
+      <p>Click here to get a random message:</p>
+
+      <button onclick="randomFact()">click for a random Gio fact.</button>
+
+      <p>Click <a href="images/images.html">here</a> to see more pictures of me!</p>
+      <p>Click <a href="blogs/blogs.html">here</a> to see what people have written about me!</p>
+      <p>Click <a href="Projects/Projects.html">here</a> to see my projects(🚧Under Construction!🚧)</p>
+
+      <p> <a href="Resume.pdf">RESUME</a>.</p>
+      <p> <a href="https://www.linkedin.com/in/giovanni-ferioli">My LinkedIn</a> </p>
+
+      <div id="fact-container"></div>
+
+      <button onclick="getName()">Use fetch()</button>
+      <div id="greeting-container"></div>
+      <div id="list-container"></div>
+
+      <h1>Leave a Message</h1>
+
+      <form action="/data" method="POST">
+
+        <p>Enter an Email:</p>
+        <textarea name="email">example@gmail.com</textarea>
+        <br/>
+
+        <p>Enter Phone:</p>
+        <textarea name="number">+1 234 567 890</textarea>
+        <br/>
+
+        <p>Enter Name:</p>
+        <textarea name="name">Giovanni Ferioli</textarea>
+        <br/>
+
+        <p>Enter subject:</p>
+        <textarea name="subject">Just to say Hi</textarea>
+        <br/>
+
+        <p>Enter message:</p>
+        <textarea name="message">Hi, How are you?</textarea>
+        <br/>
```

Comment:
Optional: You could use the [`placeholder`](https://www.w3schools.com/tags/att_textarea_placeholder.asp) attribute for the instructions or the example. E.g. instead of:
```html
<p>Enter an Email:</p>
<textarea name="email">example@gmail.com</textarea>
```
You could have:
```html
<p>Enter an Email:</p>
<textarea name="email" placeholder="example@gmail.com"/>
```
or:
```html
<textarea name="email" placeholder="Enter an email. E.g. example@gmail.com"/>
```
Or if you want to remove the example, simply:
```html
<textarea name="email" placeholder="Enter an email"/>
```
This is up to you though as this is _your_ page 😃